content/en/docs/_index.md

```diff
@@ -43,3 +43,21 @@ To build a bootable image is as simple as running `docker build`.
 - Cryptographically verified
 - instant switch from different versions
 - recovery mechanism with `cOS` vanilla images (or bring your own)
+
+## Mission
+
+The cOS-toolkit project is under the Elemental umbrella.
+
+Elemental provides a unique container based approach to define the system lifecycle of an immutable Linux derivative, without any string attached to a specific Linux distribution.
+
+At its heart, Elemental is the abstraction layer between Linux distro management and the specific purpose of the OS.
+
+Elemental empowers anyone to create derivatives from standard OCI images. Frees whoever wants to create a Linux derivative from handling the heavy bits of packaging and managing entire repositories to propagate upgrades, simplifying the entire process by using container images as base for OS.
+At the same time, Elemental provides an highly integrated ecosystem which is designed to be container-first, cloud native, and immutable.
+Anyone can tweak Elemental derivatives from the bottom-up to enable and disable its featureset.
+
+As the Elemental team, the [os2](https://github.com/rancher-sandbox/os2) project is our point of reference.
+
+`os2` is a complete derivative built with Elemental tied with the rancher ecosystem and full cycle node management solution with Kubernetes, which we are supporting directly and indirectly within changes also in the Elemental ecosystem.
+
+os2 is our main show-case, and as the Elemental team we are committed to it. It is also a show-off of how Elemental can be used to build and maintain immutable Linux derivatives: it encompasses several technologies to create a Kubernetes-focused Linux derivative which lifecycle is managed entirely from Kubernetes itself, [SDO](https://www.intel.it/content/www/it/it/internet-of-things/secure-device-onboard.html) included and automatic provisioning via cloud-init.
```

Comment:
Nit: Probably SDO reference and cloud-init are too specific here? I feel these could eventually change (I mean other standards or approaches to do the same) and the mission would still be the same.

Reply:
to my understanding they were the keys factor and requirements around os2, otherwise there would no observable (from a user perspective) difference, for example, compared to k3os.

I'm not sure if should be here indeed, but at the same time underlines the key difference between the other OSes focused on Kubernetes. 

If we don't support SDO anymore in os2, then probably should be another project.. @kkaempf @agracey, any thoughts here?

Reply:
Then we maybe have a list of less-standard components that are included in the current release